src/bridge/types.js

```diff
@@ -79,10 +79,10 @@ export interface WalletBridge<Transaction> {
   isValidTransaction(account: Account, transaction: Transaction): boolean;
 
   // render the whole Fees section of the form
-  EditFees?: React$ComponentType<EditProps<Transaction>>;
+  EditFees?: *; // React$ComponentType<EditProps<Transaction>>;
```

Comment:
pretty sure that flow will remove that semicolon

Reply:
not in a type def I think